### DISCUSSION TITLE
Is there any provision to display all albums to public without login ?

### DISCUSSION THREAD

**Discussion by Akshay-Hegde** [photoview/photoview#817] [General]
Is there any provision to display all albums to public without login ? At present each album has to be shared 

Or any possible way to auto share and list Public shared albums with cover thumbnail image ?

I followed : https://photoview.github.io/en/docs/usage-sharing/

> TODO: Explain how to share albums and media, how to set a password and how to delete them again

**Comment by viktorstrate** [comment]
This is not possible currently, but there have been some similar discussions before. See #478 #624 #174
